src/System.IO.FileSystem/tests/FileStream/ctor_sfh_fa.cs

```diff
@@ -17,17 +15,33 @@ protected virtual FileStream CreateFileStream(SafeFileHandle handle, FileAccess
         }
 
         [Fact]
-        public void InvalidHandleThrows()
+        public void InvalidHandle_Throws()
         {
-            AssertExtensions.Throws<ArgumentException>("handle", () => CreateFileStream(new SafeFileHandle(new IntPtr(-1), true), FileAccess.Read));
+            using (var handle = new SafeFileHandle(new IntPtr(-1), ownsHandle: false))
+            {
+                AssertExtensions.Throws<ArgumentException>("handle", () => CreateFileStream(handle, FileAccess.Read));
```

Comment:
Do we still have coverage for when ```ownsHandle: True```? Seems like we should leave these existing tests as one-liners that own the handle, then have your new tests test `ownsHandle: False`.